servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -219,7 +219,8 @@ private UiFileInfo getUiFileInfo(TachyonFS tachyonClient, int fileId, TachyonURI
         StorageLevelAlias storageLevelAlias =
             StorageDirId.getStorageLevelAlias(blockMeta.getParentDir().getStorageDirId());
         // The block last access time is not available. Use -1 for now.
-        uiFileInfo.addBlock(storageLevelAlias, blockId, blockSize, -1);
+        // The location information is also not available for now.
+        uiFileInfo.addBlock(storageLevelAlias, blockId, blockSize, -1, null);
```

Comment:
Yes, in this case when you click the block info from a worker it only shows id, tier, and size. So I guess I'll just change the comment to be "It's unnecessary to pass a location information here"?